src/collections/components/AuthorInfo/AuthorInfo.tsx

```diff
@@ -20,16 +19,17 @@ interface AuthorInfoProps {
  */
 export const AuthorInfo: React.FC<AuthorInfoProps> = (props): JSX.Element => {
   const { author } = props;
-  const classes = useStyles();
 
   return (
     <>
       <Typography
-        className={classes.subtitle}
         variant="subtitle2"
         color="textSecondary"
         component="span"
         align="left"
+        sx={{
+          fontWeight: 400,
+        }}
```

Comment:
i guess this is nicer/cleaner than having to make a separate file. what happens if/when we need a larger css rule set? would that go into the theme?

Reply:
@jpetto, there's a second styling option for when inline styling is taking up too much space. The JSX node would have to become its own component pretty much, e.g.

```js
const AuthorSubtitle = styled(Component)(() => ({
            fontWeight: 400,
}));
```

and then the `AuthorSubtitle` would have to be used as the pseudo HTML element instead of `Typography` directly. There's also a third-party package option, `tss-react`, which is also worth a look, but I was trying to get away with the minimum of updates here.